lutris/settings.py

```diff
@@ -23,7 +23,8 @@
 GAME_CONFIG_DIR = os.path.join(CONFIG_DIR, "games")
 
 TMP_PATH = os.path.join(CACHE_DIR, "tmp")
-BANNER_PATH = os.path.join(CACHE_DIR, "banners")
+SHADER_CACHE_DIR = os.path.join(CACHE_DIR, "shaders")
+BANNER_PATH = os.path.join(DATA_DIR, "banners")
```

Comment:
Shouldn't this be (DATA_DIR, "banners") be (CACHE_DIR, "banners")

Reply:
Whoops! I botched the merge. Honestly though, GitHub makes that awkward. I can't seem to do these merges before the PR, and the tool for doing it then is not great.